lib/network_ssl/network_ssl.c

```diff
@@ -14,6 +14,12 @@
 
 #include "network_ssl.h"
 
+/*
```

Comment:
Hmm.  I guess (given a comment you made later) that this shouldn't be a compile-time constant.  I was hoping that this was a fairly standard location...

Want to write a `network_ssl_loadroot` function?  Have it do all of the code currently in `init()`, and make other functions `assert(ctx != NULL)` -- users of this code should explicitly provide a root certificate path before trying to open any SSL connections.